src/index.js

```diff
@@ -667,7 +672,7 @@ const members = {
   },
   parentNode: {
     get () {
-      return nodeToParentNodeMap.get(this) || this.__parentNode || null;
+      return nodeToParentNodeMap.get(this) || null;
```

Comment:
Since we manually set parent node everywhere (because of WebKit) this shouldn't be necessary.